src/packages/admin-ui/src/pages/list/component.tsx

```diff
@@ -97,10 +97,26 @@ export const List = () => {
 		}
 	};
 
+	// If we have an array as a value then we need to convert this to a string to display in the table
+	const rows = (data?.result ?? []).map((row) => {
+		// Hold any overrides we need to apply
+		const overrides: { [k in keyof typeof row]?: unknown } = {};
+		const { fields } = entityByName(entity);
+		for (const key in row) {
+			const field = fields.find((field) => field.name === key);
+			if (field?.type === 'JSON') {
+				// We have an array let's stringify it so it can be displayed in the table
+				overrides[key as keyof typeof row] = JSON.stringify(row[key as keyof typeof row]);
```

Comment:
JW Can we save these types as "values"?
as in `type OverrideKey = key as keyof typeof row`

Reply:
yeah good idea!